README.md

```diff
@@ -126,6 +136,24 @@ Here you can see two important features:
 
 This way we specified which component needs to be compiled given a source file for our whole project.
 
+If you use both, multiple components and an alternate `stack.yaml` file, there is a way to specify defaults
+for the path-specific configurations.
+
+```yaml
+cradle:
+  stack:
+    stackYaml: "stack-8.3.3.yaml"
+    components:
+    - path: "./src"
+      component: "hie-bios:lib"
+    - path: "./exe"
+      component: "hie-bios:exe:hie-bios"
+    - path: "./tests/BiosTests.hs"
+      component: "hie-bios:test:hie-bios"
+    - path: "./tests/ParserTests.hs"
+      component: "hie-bios:test:parser-tests"
+```
```

Comment:
It is important to note that all `stackYaml` files *must* point to the same GHC version.
Otherwise, confusions such as https://github.com/haskell/haskell-language-server/issues/268 are bound to happen

Reply:
That's a problem that needs to be fixed upstream in the language server. It would certainly be possible to find out which ghc version is in use for the stack config (`stack [args] ghc --version`), use multiple `.stack-work` directories and load different ghcide plugins (if I understand the architecture correctly). I will add a note though with the rest of the review, for now.